app/controllers/translations_controller.rb

```diff
@@ -21,7 +21,8 @@ def all_translations
   end
 
   def render_not_found
-    @translation.errors.add(:message, 'Translation not found.')
+    @translation.errors.add(:message,
+                            'Translation not found.  Use drafts/ if you\'re looking for an unpublished translation.')
```

Comment:
Rubocop won't complain if you use double quotes in this case (so you don\'t have to quote your single quote) ;-)

It would otherwise complain about double quotes when you don't interpolate any variables, so your intentions are noble.